storm-core/src/storm.thrift

```diff
@@ -205,7 +205,8 @@ struct ExecutorSummary {
   2: required string component_id;
   3: required string host;
   4: required i32 port;
-  5: required i32 uptime_secs;
+  5: required i32 process_id;
```

Comment:
You cannot change a thrift ID like this.  It breaks backwards compatibility too much.  Please make sure that you add new tags instead of changing existing ones.  It is also usually preferable to make the new tag optional, so you can have forward and backwards compatibility.